src/signs/ui/search/MesZLContent.tsx

```diff
@@ -85,7 +87,7 @@ export default function MesZlContent({
         <div
           dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(mesZlBody) }}
         />
-        {mesZl.split('\n').length > 7 && (
+        {mesZl.split('\n').length > cutOff && cutOff !== -1 && (
```

Comment:
Avoid using magic numbers and using special values. The code is easier to understand if the domain is `number | null` vs `number` with special behaviour for some number(s), but wouldn't it be simplest just to set the default to `Infinity`?

What would happen if `cutOff` is `-2`?